pandas/core/frame.py

```diff
@@ -115,6 +115,13 @@
     side, respectively
 copy : boolean, default True
     If False, do not copy data unnecessarily
+indicator : boolean, default False
```

Comment:
Sounds good to me. I'll add, though may take me a little while (deadline for work end of the week. )